### PR TITLE
update_files

### PR DESCRIPTION
**Pull request by Manel0712**

### CODE REVIEW

File: exercise_02/getTotalWordsFromString.js [hunk 1]
```diff
@@ -3,3 +3,13 @@
  * input: string (sentence)
  * output: number
  */
+/**
+ * Calculates the words in one string;
+ * @param {string} sentence
+ * @returns number
+ */
+const getTotalWordsFromString = function (sentence) {
+  const numberWords = sentence.split(/\s+/);
```
Comment (tictools) [inline exercise_02/getTotalWordsFromString.js:12]: Do you know what does `/\s+/` mean? Which type of expression is it? 🤔

File: exercise_05/isAdditionGreaterThanFifty.js [hunk 2]
```diff
@@ -4,3 +4,17 @@
  * input: number (summand2)
  * output: boolean
  */
+/**
+ * check a pair of numbers and return true if their sum is 50 or greater than 50
+ * @param {number} summand1
+ * @param {number} summand2
+ * @returns boolean
+ */
+const isAdditionGreaterThanFifty = function (summand1, summand2) {
```
Comment (tictools) [inline exercise_05/isAdditionGreaterThanFifty.js:13]: You can simplify this logic, as far as `summand1 + summand2 >= 50` is directly returning a boolean

File: exercise_06/checkCollectionHasElements.js
Comment (tictools) [inline exercise_06/checkCollectionHasElements.js:1]: YouYou can simplify this logic, as far as `collection.length > 0` is directly returning a boolean

File: exercise_07/isTypeOf.js [hunk 3]
```diff
@@ -6,3 +6,23 @@
  * input: boolean (type)
  * output:
  */
+/**
+ * @param {string | number | boolean | array} value
+ * @param {string | number | boolean | array} type
+ * @returns {boolean}
+ */
+function isTypeOf(value, type) {
```
Comment (tictools) [inline exercise_07/isTypeOf.js:15]: This logic can be simplified, avoid using `switch` if you can use simpler structures with `if`.

**Review by tictools — CHANGES_REQUESTED** [review]

**Review by aalvar0 — APPROVED** [review]
I don't see any errors to fix.

**Review by tictools — APPROVED** [review]

**Pull request merged**